frontend/templates/product-list/sections/FilterableProductsSection/RefinementSingleSelect.tsx

```diff
@@ -137,8 +146,8 @@ const SingleSelectItem = React.memo(function SingleSelectItem({
       <HStack
          key={item.label}
          justify="space-between"
-         color={isRefined ? 'brand.500' : 'inherit'}
-         fontWeight={isRefined ? 'bold' : 'inherit'}
+         color={item.isRefined ? 'brand.500' : 'inherit'}
+         fontWeight={item.isRefined ? 'bold' : 'inherit'}
```

Comment:
Since we're not using `isRefined` anymore, does the color actually change before Algolia has finished refining the search? If not, I'd say we just drop `const [, setIsRefined] = useDecoupledState(item.isRefined);` altogether.